### DISCUSSION TITLE
Cannot exit -bash after SERVICE install

### DISCUSSION THREAD

**Discussion by BestNoob** [rejetto/hfs#732] [Q&A]
Hello,
im using Ubuntu Server. I followed the Installation instructions, installed node.js
and created the hfs.service file

But the Fileserver is not running at startup 
only thing that has changed is that i am in

-bash 

and cannot leave this ><
i tried exit, logout, cd, pushd etc.

before my system restart i was able to access the fileserver via IP

Hope that you can help me.

**Comment by rejetto** [comment]
i don't understand what you mean with "i'm in bash and cannot leave".
Try showing a screenshot.

And how do you know it's not running? did you check if "node" process is running?

**Reply by BestNoob** [reply]
> i don't understand what you mean with "i'm in bash and cannot leave". Try showing a screenshot.
> 
> And how do you know it's not running? did you check if "node" process is running?

Im at GNU bash ?!
Node.js is running

but File Server not reachable ... only after installation it was running fine ... after restart i got this issues

It doesnt matter i am logged in as user or as root its everything inside -bash and i am not able to run normal commans etc.
As you can see at the screenshot also with ROOT everything is -bash

![issue_screenshot_1](https://github.com/user-attachments/assets/a3aff342-19b4-447e-ae2a-cd47eac05754)
![issue_screenshot_2](https://github.com/user-attachments/assets/737f2c61-c17b-45a0-aeab-5ee5df05f6a6)

**Comment by rejetto** [comment]
that's ok, when you run any software as a service, you stop interacting wit it "normally". In the case of HFS, you don't see its console anymore.
I'm don't know linux and I don't know if you could access it in some way, but that's not supposed to be a problem.
You should interact with HFS, when it works, accessing it with your browser, http://localhost:PORT
If something is going wrong, you probably will see errors in a file called "debug.log" inside folder ".hfs" (with the dot in front), same folder where your configuration is saved (config.yaml file)

**Reply by BestNoob** [reply]
Everything now is the relative path also for executing and some commands are not working xD 
everything and all now is enterpreted by node bash

Hope you can help me out of this ....

I am myself new to linux i prefer windows servers but .. i wanted to learn and use some features of ubuntu
and ubuntu servers are much cheaper (because u dont need a win licence)

But for some ppl. there is a rule we are HUMAn so we need a HUD xD

**Comment by rejetto** [comment]
i don't understand, did you find the folder ".hfs" that i said?

**Reply by BestNoob** [reply]
Yes everything is working. When i leave the BASH mode with strg (ctrl) + C 

unloaded plugin antibrute

and HFS is no longer working but i created the 

hfs.service 

file ....

**Reply by BestNoob** [reply]
This was my setup..... Ubuntu 22.04

sudo apt install npm
npm install -g node-gyp

# installs fnm (Fast Node Manager)
curl -fsSL https://fnm.vercel.app/install | bash

# activate fnm source (use /user/.bashrc)
source ~/.bashrc

# download and install Node.js
fnm use --install-if-missing 18

# verifies the right Node.js version is in the environment
node -v # should print `v18.20.4`

# verifies the right npm version is in the environment
npm -v # should print `10.7.0`

npx hfs@latest

create-admin PASSWORD

sudo systemctl daemon-reload && sudo systemctl enable hfs && sudo systemctl start hfs && sudo systemctl status hfs
